reference/docs-conceptual/PowerShell-Support-Lifecycle.md

```diff
@@ -24,13 +23,30 @@ guarantee there that the community will address or resolve your issue in a timel
 have a problem that requires immediate attention, you should use the traditional, paid support
 options.
 
-## Lifecycle of PowerShell Core
+## LifeCycle of PowerShell 7
+
+With the release of PowerShell 7, support has moved from the Microsoft Modern Lifecycle Policy to follow [.NET Core's support lifecycle][Long-Term].
```

Comment:
So we actually are still on the Modern Lifecycle Policy, but that's an umbrella term for Microsoft products that are on more fix-forward lifecycles and where customers need to refer to each individual policy in order to ascertain whether they're supported. For instance, [.NET Core is also Modern Lifecycle Policy](https://support.microsoft.com/en-us/lifecycle/search?alpha=.NET%20Core)

I'd use something more along the lines of: "With the release of PowerShell 7, PowerShell continues to be supported under the [Microsoft Modern Lifecycle Policy](https://support.microsoft.com/en-us/help/30881/modern-lifecycle-policy) with support dates driven by the [.NET Core's support lifecycle][Long-Term]." 

Not suuuuper happy with my wording, so feel free to wordsmith, but you get the gist.

Reply:
Updated but won't resolve unless you think it's a good representation.

Reply:
LGTM